jina/types/sets/document_set.py

```diff
@@ -88,3 +88,21 @@ def build(self):
 
     def sort(self, *args, **kwargs):
         self._docs_proto.sort(*args, **kwargs)
+
+
+class MultiModalDocumentSet(DocumentSet):
```

Comment:
@JoanFM We need a consistent naming here. `MultiModalDocumentSet` and `MultimodalDocument` is not a good choice.

Reply:
Oh yes I will change it, what is preferred?

Reply:
Personally,  `MultimodalDocumentSet ` is prefered because `multimodal` is one word.

Reply:
In case, we might have a `CrossModalDocumentSet` in the future, I'd prefer `MultiModalDocumentSet`.

Reply:
I think there will not be a CrossModalDocumentSet

Reply:
Opened a PR to fix this #1390